docs/reference/gedcom-date-formats.md

```
@@ -0,0 +1,1125 @@
+```
+
+Any dates in the gap should be flagged as impossible.
+
+**Other transitions**: Russia 1918, Greece 1923, etc. (see section 4.2)
+
+### 8.3 B.C./B.C.E. Dates
+
+**GEDCOM Format**: Year can be negative or use "B.C." suffix
+
+**Examples**:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)